lib/doorman.ex

```diff
@@ -1,2 +1,48 @@
 defmodule Doorman do
+  @moduledoc """
+  Provides helpers that take advantage of the options configured in your config
+  files.
+  """
+
+  @doc """
+  Authenticates a user by their email and password. Returns true user is found
+  and the password is correct, otherwise false.
+
+  Requires `user_module`, `secure_with`, and `repo` to be configured via
+  `Mix.Config`.
+  """
+  def authenticate(email, password) do
+    user = repo_module.get_by(user_module, email: email)
+    user && authenticate_user(user, password)
+  end
+
+  @doc """
+  Authenticates a user. Returns true user is found and the password is
```

Comment:
Also it might be helpful to illustrate the difference between those help and the other one. It's possible to deduce, but would be helpful to have something like: "Use `authenticate/2` if you would like to authenticate with email and password"?